README.md

```diff
@@ -148,6 +148,7 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | [7egend](https://www.7egend.cr/) [:rocket:](https://jobs.7egend.cr/) | Digital Agency with focus on Web, Mobile and IoT. | `Lisboa` `Porto` |
 | [Bliss Applications](https://www.blissapplications.com/) [:rocket:](https://www.blissapplications.com/careers) | UX/UI & software-driven company. | `Lisboa` `Porto` |
 | [Coletiv](https://www.coletiv.com) [:rocket:](https://coletiv.com/#jobs) | Mobile and backend development. | `Porto` |
+| [Deemaze](https://deemaze.com/) [:rocket:](https://deemaze.com/contacts) | Web, Mobile development (RoR, Elixir, React, Vue, Swift, Kotlin). | `Braga` `Coimbra` |
```

Comment:
The rocket link is mean for careers page, as in, job posts listings. 

Deemaze does not seem to have one so this should be deleted. As alternative, use linkedin jobs or medium tags for job posts as I think that would be an OK link to use.

Please either remove the rocket link or change into one which contains job posts.

Reply:
I would prefer a link like this https://medium.com/deemaze-software/tagged/jobpost so if in the future a new job-post post is added the link would list all medium posts about job-posts

Still this seems OK to me for now.